dev/Scroller/InteractionTests/ScrollerTestsWithInputHelper.cs

```diff
@@ -1183,27 +1225,37 @@ public void PanTowardsTwoManditoryIrregularSnapPoint()
         }
 
         [TestMethod]
-        [TestProperty("Description", "Apply a single mandatory Regular snap point across the extent, with Offset equal to Start, and pan within it.")]
-        public void PanWithinARegularMandatorySnapPoint()
+        [TestProperty("Description", "Apply a single mandatory repeated snap point across the extent, with Offset equal to Start, and pan within it.")]
+        public void PanWithinARepeatedMandatorySnapPoint()
         {
-            PanWithinARegularMandatorySnapPoint(withOffsetEqualToStart: true);
-        }
+            if (PlatformConfiguration.IsOSVersionLessThan(OSVersion.Redstone5))
+            {
+                Log.Warning("This test relies on touch input, the injection of which is only supported in RS5 and up. Test is disabled.");
+                return;
+            }
 
-        [TestMethod]
-        [TestProperty("Description", "Apply a single mandatory Regular snap point across the extent, with Offset different from Start, and pan within it.")]
-        public void PanWithinARegularMandatorySnapPointWithDifferentOffset()
-        {
-            PanWithinARegularMandatorySnapPoint(withOffsetEqualToStart: false);
+            PanWithinARepeatedMandatorySnapPoint(withOffsetEqualToStart: true, alignmentIndex: 0); // Near alignment
```

Comment:
Did you want to just provide an enum for this alignment thing rather than re-comment across these calls and the use sites?

Reply:
Yes. Done.